lib/m2ts/caption-stream.js

```diff
@@ -515,6 +515,14 @@ Cea608Stream.prototype.flushDisplayed = function(pts) {
   var content = this.displayed_
     // remove spaces from the start and end of the string
     .map(function(row) {
+      if (row === undefined || row === null) {
```

Comment:
Actually, since we control `this.row_`, it (and caption data) should never fall outside the valid range -- provided this whole "conflicting base row and roll-up" is handled correctly.

Reply:
`undefined` was originally possible due to the indexing errors if `topRow` would end up being < 0. `null` might be overkill. 

I assumed that it would still be possible for exceptions to be thrown on invalid data so I also added this check.  I could alternatively use `try/catch`, since that would avoid any invalid data exceptions

Reply:
Yeah, but now that `topRow` (or any row indexer) will never be outside the range 0-14, I don't think this could ever trigger.

Reply:
I'll switch this to have a `try/catch` then